### PR TITLE
feat(dashboard): support dashboard

### PR DESCRIPTION
**Pull request by CHLK**
#### What type of PR is this?
type-feature

#### What this PR does / why we need it:
- feat(resourceLastAccess): add batchDeleteResourceIds to ResourceLastAccessService
- feat(ResourceType): add Chart and Dashboard to ResourceType
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/resourcehistory/ResourceLastAccessService.java [hunk 1]
```diff
@@ -68,6 +69,18 @@ public ResourceLastAccessEntity add(Long organizationId, Long projectId, Long us
         return entity;
     }
 
+    public int batchDeleteResourceIds(Long organizationId, Long projectId, ResourceType resourceType,
```
Comment (yhilmare) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/resourcehistory/ResourceLastAccessService.java:72]: `@NonNull` annotation may be better?
why should we judge null if we have done it at the `server/odc-service/src/main/java/com/oceanbase/odc/metadb/resourcehistory/ResourceLastAccessRepository.java`

File: server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/ResourceType.java [hunk 2]
```diff
@@ -143,4 +145,13 @@ public String getLocalizedMessage() {
         Locale locale = LocaleContextHolder.getLocale();
         return translate(null, locale);
     }
+
+    public static Optional<ResourceType> getByName(String name) {
```
Comment (yhilmare) [inline server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/ResourceType.java:149]: what's the difference between this and `ResourceType.valueOf()`?

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

File: server/odc-core/src/main/resources/i18n/BusinessMessages.properties [hunk 3]
```diff
@@ -86,6 +86,9 @@ com.oceanbase.odc.ResourceType.ODC_GIT_REPOSITORY=Git Repository
 com.oceanbase.odc.ResourceType.ODC_WORKSHEET=Worksheet
 com.oceanbase.odc.ResourceType.ODC_WORKSPACE=Workspace
 com.oceanbase.odc.ResourceType.ODC_WORKFLOW_CLUSTER=Workflow Cluster
```
Comment (JessieWuJiexi) [inline server/odc-core/src/main/resources/i18n/BusinessMessages.properties:88]: 这个数仓开发术语讨论的时候定的是 Compute Node Pool

Comment (CHLK) [reply server/odc-core/src/main/resources/i18n/BusinessMessages.properties]: okay,  I'll make this change.

**Review by JessieWuJiexi — APPROVED** [review]

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**